### PR TITLE
 Fix incorrect normalization values for train and validation loss

### PR DESCRIPTION
**Pull request by jcardonnet**
Normalizing train and validation loss using len(train_loader.dataset) and len(valid_loader.dataset) results in computed validation about 4x lower than the training loss, even after 20 epochs, which is clearly a red flag.
This is due to  len(train_loader.dataset) and len(valid_loader.dataset) being 5000 rather than 4000 and 1000 respectively, as they should.
Normalizing train and validation losses using len(train_loader.dataset) and len(valid_loader.dataset) respectively solves this inconsitency.

### CODE REVIEW

**Comment by cezannec** [comment]
Thank you!
